src/components/Filter/Filter.css

```diff
@@ -39,6 +39,11 @@
     left: 0;
     width: 100%;
   }
+
+  .inputs[aria-expanded="false"] {
```

Comment:
It's ironic that we are using ARIA attributes for accessibility... but display: none; makes the content not accessible for screen readers.

Should we create a util class for "hiding"? So we stop using `display: none`

Reply:
I'm not sure I understand. When the inputs are hidden we don't want them to be accessible, isn't?